ddtrace/sampler.py

```diff
@@ -175,11 +200,15 @@ def sample(self, span):
                 matching_rule = rule
                 break
         else:
-            # No rule matches, use the default sampler
+            # If no rules match, use our defualt sampler
+            # This may be a RateByServiceSampler
             matching_rule = self.default_sampler
 
         # Sample with the matching sampling rule
-        span.set_metric(SAMPLING_RULE_DECISION, matching_rule.sample_rate)
+        # Only set if it isn't the default RateByServiceSampler
+        #   since that gets it's own metric
+        if isinstance(matching_rule, SamplingRule):
```

Comment:
Rate limiter is only meant to be applied to Rules, as setting 100% of rate might generate uncontrolled traffic.
Rate by service gives a number of trace/s controlled in the agent, this is controlled and always under a fix number/s

Reply:
updated